### PR TITLE
[Merged by Bors] - feat: self_lt_pow

### PR DESCRIPTION
**Pull request by mo271**
---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

File: Mathlib/Algebra/GroupPower/Order.lean [hunk 1]
```diff
@@ -526,6 +526,11 @@ theorem pow_lt_self_of_lt_one (h₀ : 0 < a) (h₁ : a < 1) (hn : 1 < n) : a ^ n
     _ = a := pow_one _
 #align pow_lt_self_of_lt_one pow_lt_self_of_lt_one
 
+theorem self_lt_pow_of_one_lt (h : 1 < a) (h2 : 1 < m) : a < a ^ m := by
```
Comment (ADedecker) [inline Mathlib/Algebra/GroupPower/Order.lean:529]: Judging by the names in this file it seems that the versions assuming `1 < a` don't have `of_one_lt` in their name (see [pow_lt_pow](https://leanprover-community.github.io/mathlib4_docs/Mathlib/Algebra/GroupPower/Order.html#pow_lt_pow) VS [pow_lt_pow_of_lt_one](https://leanprover-community.github.io/mathlib4_docs/Mathlib/Algebra/GroupPower/Order.html#pow_lt_pow_of_lt_one)). Could you rename this lemma to `self_lt_pow` and move it before `strictAnti_pow` (to keep both cases loosely separate)?

**Review by ADedecker — COMMENTED** [review]
Thanks!

bors d+

File: Mathlib/Algebra/GroupPower/Order.lean (hunk 1 above)
Comment (mo271) [reply Mathlib/Algebra/GroupPower/Order.lean]: done, thanks!

**Pull request closed without merging**